backend/QA/src/main/java/com/example/qa/dao/question/QuestionDao.java

```diff
@@ -219,8 +219,9 @@ public void update(
       long createdAt,
       long updatedAt,
       int numAnswer,
+      QuestionRecord record,

```

Comment:
Did the transaction update also remove some values?
I think the `record` isn't required for `transaction` on Cosmos DB, too.

We have the issue of updating when an application doesn't use Scalar DB transaction.
(Scalar DB adds the non-updated values for updating in a transaction.)

Reply:
> Did the transaction update also remove some values?

No.
I will remove this change with the new `createPutWithForUpdate` method.

Reply:
I think so. And, it is better to add a comment on why the read record is needed for `storage`. It will be helpful to delete the modification if Cosmos DB would support the partial updating.
So, when I commented about adding `get`, I thought adding get in the `update` method. I know it's a bit redundant, but it would be easier to delete it.

Reply:
@yito88 san,
So, Can we use the same `createPutWithForUpdate` method for both `storage` and `transaction`?

Reply:
@yito88 san,
Thank you for the detailed explanation.
The source code modified accordingly.